### DISCUSSION TITLE
Batch Hardcode Subtitles

### DISCUSSION THREAD

**Discussion by kuzon3** [staxrip/staxrip#941] [Ideas]
If possible can you add a option to batch hardcode subtitles?

**Comment by jessielw** [comment]
There doesn't seem to be anyone maintaining the project atm. However, batching many files with hard coded subs in a GUI could deem pretty difficult. 

This might be something you want to do via scripting.

**Reply by iamsrk48** [reply]
can be possible if filename are same staxrip will be auto detect file names and than pick subtitles for every videos
